### PR TITLE
Typing: ppe.onnx.export_testcase

### PR DESCRIPTION
**Pull request by asi1024**
~Blocked by #470.~

### CODE REVIEW

**Comment by asi1024** [comment]
Rebased.

File: pytorch_pfn_extras/onnx/export_testcase.py [hunk 1]
```diff
@@ -79,7 +87,12 @@ def strip_cmd(cmd):
     return ret
 
 
-def _export_util(model, args, f, **kwargs):
+def _export_util(
+        model: torch.nn.Module,
+        args: Tuple[Any, ...],
```
Comment (take-cheeze) [inline pytorch_pfn_extras/onnx/export_testcase.py:92]:
```suggestion
        args: Sequence[Any, ...],
```

Comment (take-cheeze) [reply pytorch_pfn_extras/onnx/export_testcase.py:92]: For compatibility with [torch.onnx.export](https://pytorch.org/docs/stable/onnx.html#torch.onnx.export) maybe `Union[Sequence[Any, ...], torch.Tensor]` might be better.
Sometimes list would be also passed so allowing it with `Sequence` would be better

Comment (asi1024) [reply pytorch_pfn_extras/onnx/export_testcase.py]: Fixed in 5058f03! Thanks!

**Comment by asi1024** [comment]
Rebased.

**Review by take-cheeze — APPROVED** [review]
👍

**Pull request merged**